lib/balrog/middleware.rb

```diff
@@ -56,7 +60,9 @@ def handle_login(env)
     end
 
     if @password_hash == submitted_password
-      env['rack.session'][:balrog] = 'authenticated'
+      balrog_cookie = { value: 'authenticated' }
```

Comment:
It's not _really_ a cookie is it, it's the session object. This name is a little confusing, perhaps call it something different?